tests/system.py

```diff
@@ -657,6 +657,13 @@ def tearDown(self):
         for table in self.tables_to_delete:
             table.delete()
 
+    def _maybe_emulator_skip(self, message):
```

Comment:
I'd rather give it a different name, e.g., `_skip_if_emulator` or so.

Reply:
@mf2199  As you can see here to maintain consistency i put the same name of method https://github.com/q-logic/python-bigtable/blob/0134f2edfb0893260feaad31dec27cc5962914b7/tests/system.py#L835-L839